docs/Operation-Architecture/Orbit_UI/Using_Orbit.rst

```diff
@@ -2,33 +2,25 @@ Using Orbit
 ===========
```

Comment:
too short ... does it make sense to merge with other files?

Reply:
This serves the purpose of introducing a chapter in a book as well as directing navigation all its sub-topics in the web site. it is logically "superior" to the other topics, and it has no peers. I am inclined to leave well enough alone for now. This is not broken. We should revisit it after the RST is merged and published. In the meantime, this is the great being the enemy of the good.